src/app/components/mnoe-api/product-instances.svc.coffee

```diff
@@ -7,7 +7,9 @@ angular.module 'mnoEnterpriseAngular'
 
     productInstancesPromise = null
     @getProductInstances = ->
-      return productInstancesPromise if productInstancesPromise?
+      # Note: Temporarily commented out product instance cache till actual
+      #       connec implementation from APPINT-1238 for sync status is done.
+      # return productInstancesPromise if productInstancesPromise?
```

Comment:
I'm actually not completely sure about this part, even when APPINT-1238 is complete. Having this promise will mean that any time the status is updated, it will be out of sync until the page is refreshed. Not sure of the performance implications, but it may just be better to have this removed. @ouranos Thoughts on this?

Reply:
Looks like this is only called on the App Management page.
So I wouldn't be shocked to have the page reload status (especially when we have a loader)
This pattern was mostly done for the `AppInstances` in the dock who took a while to load

Btw there's also `LocalStorage` caching.

Reply:
Ah, removing the LocalStorage caching as well.